.github/workflows/release.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
 
       - name: Set branch name and changeset PR title
         run: |
-          echo "CHANGESET_PR_TITLE=$(echo "ci(changesets): versioning packages for branch \`${{ github.ref_name }}\`")" >> $GITHUB_ENV
+          echo "CHANGESET_PR_TITLE=$(echo "ci(changesets): versioning packages - `v${{ env.BUILD_VERSION }}` @ \`${{ github.ref_name }}\`")" >> $GITHUB_ENV
```

Comment:
From:
 - ci(changesets): versioning packages for branch `master`

To:
 - ci(changesets): versioning packages - `v0.78.1` @ `master`